pyvista/examples/downloads.py

```diff
@@ -182,8 +182,16 @@ def download_blood_vessels():
     mesh.set_active_vectors('velocity')
     return mesh
 
-def download_iron_pot():
-    """Download iron pot dataset."""
+def download_iron_pot():  # pragma: no cover
+    """Download iron protein dataset.
+
+    DEPRECATED: Please use ``download_iron_protein``.
+
+    """
+    return self.view_isometric()
```

Comment:
Oh, crap, just having read your comment first I didn't understand how the non-methodness came into play.

I copied that deprecation notice from a deprecated method, and I managed to copy over the return rather than calling the new function as I meant to. Ugh, apologies.